app/models/user.rb

```diff
@@ -184,9 +184,11 @@ def self.send_alert_membership_will_expire(config, log)
       if user.membership_current?
```

Comment:
This doesn't belong here.  Doesn't `MemberExpireAlert` take care of this now?

Reply:
That's been removed.